docs/docs/Development.md

```diff
@@ -6,13 +6,13 @@ description: >
   How to develop for triggers and functions
 ---
 
-To aid in the development of new libraries of triggers and functions you can use the type declaration files for the triggers and functions API. This can be used by your preferred development environment to provide autocompletion and type checking based on the type information. You can install it using the following command:
+To aid in the development of new libraries of triggers and functions, you can use the type declaration files for the [triggers and functions API](/docs/interact/programmability/triggers-and-functions/concepts/javascript_api/), which allows your preferred development environment to provide autocompletion and type checking. You can install this information using the following command:
 
 ```bash
 npm install https://gitpkg.now.sh/RedisGears/RedisGears/js_api
 ```
 
-Or add it directly as a dependency to your `package.json`:
+Or you can add it as a dependency to your `package.json` file:
```

Comment:
We should specify devDependency, as it is only used during development and we do not want to bundle it when uploading the library.